common/web/gesture-recognizer/src/tools/host-fixture/extract-fixture.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+#
+# Extracts and returns the fixture contents of `host-fixture.html`.
+#
+
+set -eu
+
+# This script operates from the perspective of its own directory.
+THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")"
+cd $(dirname "$THIS_SCRIPT")
+
+FIXTURE_START="    <!-- START: recognizer host fixture -->"
+FIXTURE_END="    <!-- END: recognizer host fixture -->"
+
+sed -n "/$FIXTURE_START/, /$FIXTURE_END/{ /$FIXTURE_START/! { /$FIXTURE_END/! p } }" host-fixture.html
```

Comment:
... and the differences between Mac `sed` (and `awk`) strike again.

GNU awk is possible to get on macOS via Homebrew, but that _would_ mean more build-agent setup.

Alternatively... a small headless TS program should be able to handle this.  Maybe I'll just take that approach instead, since we've already got Node in place anyway.

Reply:
And, done.  Fixture extraction + insertion each have their own node script now.

The exact setup may not be optimized, but it's at least "good enough".